app/src/main/java/com/x8bit/bitwarden/data/auth/repository/AuthRepositoryImpl.kt

```diff
@@ -298,6 +302,7 @@ class AuthRepositoryImpl(
                     isBiometricsEnabledProvider = ::isBiometricsEnabled,
                     vaultUnlockTypeProvider = ::getVaultUnlockType,
                     isDeviceTrustedProvider = ::isDeviceTrusted,
+                    firstTimeState = UserState.defaultFirstTimeState,
```

Comment:
Can we make an extension on `AuthDiskSource` for this too. That's what we do for some of the others above ^^